configs/webpack-config-compass/src/index.ts

```diff
@@ -67,6 +67,11 @@ const sharedResolveOptions = (
       // Additionally `ampersand-sync` brings into the bundle a number of other dependencies
       // that are outdated and having known vulnerabilities.
       'ampersand-sync': false,
+      // `jose` provides a browser export that uses webcrypto APIs and returns
+      // webcrypto objects to represent keys, but openid-client requires
+      // KeyObject instances from the Node.js crypto API (https://tinyurl.com/2rrtu2hy).
```

Comment:
Why do we have this as a tinyurl?

Reply:
It’s literally just because it’s a long URL that’s wider than most comments here. :slightly_smiling_face:  I don’t really care, happy to put the full link here as well :)